Source/Config/Frames/Selling.xml

```diff
@@ -37,32 +37,57 @@
           <Anchor point="TOPLEFT" relativeKey="$parent.AuctionChatLog" relativePoint="BOTTOMLEFT" />
         </Anchors>
       </Frame>
-      <Frame inherits="AuctionatorConfigurationCheckbox" parentKey="AltClick">
-        <KeyValues>
-          <KeyValue key="labelText" value="AUCTIONATOR_L_CONFIG_ALT_CLICK" type="global" />
-          <KeyValue key="tooltipTitleText" value="AUCTIONATOR_L_CONFIG_ALT_CLICK_TOOLTIP_HEADER" type="global" />
-          <KeyValue key="tooltipText" value="AUCTIONATOR_L_CONFIG_ALT_CLICK_TOOLTIP_TEXT" type="global"/>
-        </KeyValues>
+      <Frame inherits="AuctionatorDropDown" parentKey="BagSelectShortcut">
+        <Scripts>
+          <OnLoad>
+            self.text = {"Right Click", "Alt + Left Click", "Alt + Right Click", "Shift + Left Click", "Shift + Right Click", "Disabled"}
```

Comment:
Can we move this into the mixin? I like to keep code out of the UI files...

Reply:
I needed a way to input the contents of the dropdown into the component, and the only way to do an array is to type it into an OnLoad handler.

Reply:
Found a way around the problem, store the array as a string, with the items separated by `;`, gets rid of the code in the XML 😃